frontend/src/components/SearchResultInfo.tsx

```diff
@@ -2,19 +2,19 @@ import { Link } from 'react-router-dom';
 
 type Props = {
   total: number;
-  city?: string;
+  city: string;
 };
 
 const SearchResultInfo = ({ total, city }: Props) => {
   return (
-    <div className="text-xl font-bold flex flex-col gap-3 justify-between lg:items-center lg:flex-row lg-flex-row">
+    <div className="text-xl font-bold flex flex-col gap-3 justify-between lg:items-center lg:flex-row">
       <span>
-        {total} Restaurant found in {city}
+        {total} Restaurants found in {city}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/39#issuecomment-2406845576>#a6e538</a><span> - 10/11/2024, 08:17 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The pluralization of 'Restaurants' is now always applied, which may be incorrect when there's only one result.
</div>
<div id="fix">
<b>Fix:</b> Implement conditional rendering to use singular 'Restaurant' when total is 1, and plural 'Restaurants' otherwise.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12 +12,3 @@
 -        {total} Restaurants found in {city}
 +        {total} {total === 1 ? 'Restaurant' : 'Restaurants'} found in {city}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged